test/vms/ramen/etc/init.d/increase-root-filesystem-size

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+LOGFILE="/var/log/resize-filesystem.log"
+
+case "$1" in
+    start)
+        echo "Starting resize..." | tee -a "$LOGFILE"
+        df -h / >> "$LOGFILE"
+        DEVICE=$(df -h / | grep -vE '^Filesystem|tmpfs|cdrom' | awk '{print $1}')
+        echo "Device: $DEVICE" | tee -a "$LOGFILE"
+        resize-filesystem "$DEVICE" 2>&1 | tee -a "$LOGFILE"
```

Comment:
We e have an easy way to tell if resize is needed? I don't want to delay startup for unneeded resize.

Getting the device size is easy with lsblk, not sure about getting the filesystem size, maybe df / with some flags can make it easy.

Reply:
done